static/app/views/issueList/overview.polling.spec.jsx

```diff
@@ -65,21 +62,17 @@ describe('IssueList -> Polling', function () {
       },
     };
 
-    wrapper = mountWithTheme(
+    render(
       <OrganizationContext.Provider value={TestStubs.Organization()}>
         <IssueList {...newRouter} {...defaultProps} {...p} />
       </OrganizationContext.Provider>,
-      routerContext
+      {context: routerContext}
     );
 
     await act(async () => {
```

Comment:
Is this still necessary? (the await act?)

Reply:
Yeah looks like we don't need the act, but need the tick.

I'll follow up